### ISSUE TITLE
Problems with Audio Capture in windows

### ISSUE THREAD

**Issue by ChrisLeh** [pupil-labs/pupil#1453]
Hi all,
when I want to record audio (Audio Capture) I have one of the following problems:
1. Sometimes I cannot select the Audio Source, although it's shown as an option in the dropdown.
2. When I can select the source, pupil capture crashes ("No respond"), when clicking on "record".
3.  When playing a record with audio-capturing (for the rare times this worked), Pupil player crashes ("No respond") 

Please be aware, that I just began using pupil labs and my technical knowledge is not too big.

If this helps, here some informations on my computer:
Windows 10 Enterprise, Intel i7-7820HQ CPU@2,90GHz, 16,0 GB RAM

Thanks for your help,
- Christoph

**Comment by papr** [comment]
Hi @ChrisLeh 
- What microphone do you use for recording (1./2.)?
- Also, could you please share the recording (3.) which crashes Pupil Player for you?

**Comment by ChrisLeh** [comment]
Hey papr,
thanks for your answer!

1./2.: I tried different audio sources: (internal microphone, Plantronics Blackwire 325.1). In the dropdown I chose either primary source or the specific device. Both options broke down the program when recording.

3.: I have to create a file new "neutral" file which breaks down the pupil player and hand it later. My existing ones contain confident contents.

Best regards
Christoph
